.travis.yml

```diff
@@ -6,6 +6,7 @@ otp_release:
 sudo: true
 before_script:
   - sudo pip install crossbar
+script: "mix test && mix spell.example.pubsub"
```

Comment:
From what I read in travis website, the result of `script:`  is the one used to decide if the build was successful or not. 
If I add it to the after script I don't know if we get that information.